src/__tests__/network.integration.ts

```diff
@@ -0,0 +1,54 @@
+import { Message } from "../message"
+import { Base, Network } from "../network"
+import { ID1, SERVERS } from "./data"
+
+describe("Network", () => {
+  const HEX = "d28440a053d92711a2036673746174757305c11a64234ee740"
```

Comment:
This request is well-formed but will return a Many error because it's out of the accepted time bounds. It's not useful to test the content of the response, just that we received one.